cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -82,7 +79,7 @@ spec:
           - /monitor
           - --stackdriver-prefix={{ prometheus_to_sd_prefix }}/addons
           - --api-override={{ prometheus_to_sd_endpoint }}
-          - --source=fluentd:http://localhost:31337?whitelisted=stackdriver_successful_requests_count,stackdriver_failed_requests_count,stackdriver_ingested_entries_count,stackdriver_dropped_entries_count
+          - --source=fluentd:http://localhost:24231?whitelisted=stackdriver_successful_requests_count,stackdriver_failed_requests_count,stackdriver_ingested_entries_count,stackdriver_dropped_entries_count
```

Comment:
What's the impact of changing this port? Is this customer visible or mainly just used by K8s built-in monitoring?

Reply:
I just left a comment on the PR asking the same thing for @crassirostris. This is public facing in the 1.7 and 1.8 change logs, but waiting for Mik's input.

Reply:
Sounds good.

Reply:
The latter, the change in fine